js/script.js

```diff
@@ -25,4 +25,6 @@ function closeMenu() {
 
 closeIcon.addEventListener('click', closeMenu);
 
-
+menuItem.forEach((item) => {
+    item.addEventListener('click', closeMenu);
+  });
```

Comment:
- [ ] Good job so far 👍 but currently your hamburger menu is unresponsive. Kindly address this issue and verify that the mobile menu works as expected before submitting it for review.

Reply:
@Kingstalux  I am cannot lose because it is responsive. can you please through more light on it or can we be on  a call

Reply:
And this was not part of the project description
@Kingstalux I am lost because it is responsive. can you please through more light on it or can we be on a call